### ISSUE TITLE
Transaction value shown as sum of all outputs being used right after sending it

### ISSUE THREAD

**Issue by vp1111** [mymonero/mymonero-app-js#275]
Today I noticed that right after I sent 1 XMR the transaction screen was showing a value of 4.3 XMR. I think the transaction maybe wasn't still in the queue or something, but it showed what I believe was the sum of all outputs being used, including what would be the change. After a couple of seconds when I left the screen and came back it showed the correct value of -1 XMR. I believe it's a bug.

**Comment by paulshapiro** [comment]
yeah this is a long standing server thing related to the tx not having made it into a block yet. fixing soon.
